akka-docs/rst/scala/stream/stages-overview.rst

```diff
@@ -84,6 +84,23 @@ If the future fails the stream is failed with that exception.
 
 **completes** after the future has completed
 
+fromFutureSource
+^^^^^^^^^^^^^^^^
+Starts a new `Source` from another `future` source.
```

Comment:
Skip this first sentence, as it is different from the form of the other descriptions in the file (and also is incorrect, the stage is a source, it does not start anything).

"Streams the elements of the given future source once it successfully completes. If the future fails the stream is failed."